src/main/java/dinkplugin/message/Fieldable.java

```diff
@@ -0,0 +1,7 @@
+package dinkplugin.message;
+
+import java.util.List;
+
+public interface Fieldable {
+    List<Field> getFields();
+}
```

Comment:
Might be Java noob, but the Fieldable interface feels unnecessary to me when we have `NotificationData` as a base class already, since it's a base class that only implements a single interface